modules/insights-operator-what-information-is-collected.adoc

```diff
@@ -12,5 +12,6 @@ The following information is collected by the Insights Operator:
 * Errors that occur in the cluster components
 * Progress information of running updates, and the status of any component upgrades
 * Details of the platform that {product-title} is deployed on, such as Amazon Web Services, and the region that the cluster is located in
+* Cluster workload information transformed into discreet SHA (Secure Hash Algorithm) values, allowing Red Hat to assess workloads for security and version vulnerabilities without disclosing sensitive details.
```

Comment:
Our style guide says to put the full form first and keep the ending punctuation parallel.

```suggestion
* Cluster workload information transformed into discreet Secure Hash Algorithm (SHA) values, which allows Red Hat to assess workloads for security and version vulnerabilities without disclosing sensitive details
```

Reply:
Thanks @kalexand-rh changes pushed.